### ISSUE TITLE
Rotation breaks RS-Latch functionality

### ISSUE THREAD

**Issue by Hirsu** [amadornes/SCM-Localization-Issues#260]
When using a blueprint to create a RS-Latch circuit, the circuit works fine out-of-the-box. However, when a screwdriver is used to rotate the circuit, the circuit no longer performs as it should. Instead, when an input pulse is received, the output signal stays on. The only way to reset the circuit is by manually switching the lever on the tile. 

I've made a short Youtube video demonstrating this issue: [click](https://www.youtube.com/watch?v=VBRazNnI7ig&t=1s).

I'm using the latest All the Mods modpack (v1.47) which uses Forge 2151 and Super Circuit Maker 1.0.4. I've also removed Infernal Mobs and added a few mods:

- Optifine (the issue persists even after removing Optifine)
- Nature's Compass
- Login Shield
- Literal Ascension (+ dependencies)
- Akashic Tome
- Dynamid Surroundings Unofficial
- Sound Filters and Sound Cap
- Neat

I'm the same person who also reported the previous issue regarding the problem with Optifine. This one persists even after removing said mod, so I don't believe Optifine is the culprit here. Still, I apologise for bugging you with two issues in a row ;) I hope these reports are of any use in the development of SCM.

Thanks!

**Comment by gDKdev** [comment]
It's a bug causing outputs to behave weird, you need to detach and attach them again (maybe also only update work too, eg place something next to them). I don't know why it happens exactly, but I can say that it's not a fault of another mod, because I can play only mc+scm.

**Comment by Hirsu** [comment]
Wait, I'm a bit confused, so you're saying it's not a cross-mod interaction? As in it's a bug within SCM itself?

Also, I just tried your suggestion and reconnected all the inputs and outputs after having rotated a state cell. The issue still persisted. Same goes for forcing a block update by placing new blocks around the circuit after the rotation. I might be doing something wrong here; if so, could you be a bit more specific about what I'm supposed to do?

**Comment by gDKdev** [comment]
Yes, I said it's a bug within SCM itself. Your issue is a bit different than the one i know of... I never had the issue that a connection is formed on rotation.

**Comment by Hirsu** [comment]
Well, if you take a look at my second video, you'll notice I'm not imagining things :)
Guess I found a legitimate bug \o/.

**Comment by Xiaminou** [comment]
Yep, I can confirm the bug is still present, the lever is not flipping from the timer pulse for some reason. I even tried replacing some of the wires and pieces but nothing fixes it.
